### PR TITLE
fix: Add project based tailwind config file generation

### LINKED ISSUE #8: Generates a duplicate tailwind config file
Hello,

I'm working on a Typescript project with a `tailwing.config.ts` file. I used the cli to set up Tremor, and it created a `tailwind.config.js` file along the existing .ts file. Maybe the cli tool should check if the project is in TS or JS, and if a config.ts file already exists?

### PR DESCRIPTION
**Pull request by rajdip-b**
## Changes
- Changes made in `./src/index.ts`
- Feature to generate `tailwind.config.js` or `tailwind.config.ts` based on project setup
- Approval to overwrite a tailwind config in case it already exists.

### CODE REVIEW

**Comment by rajdip-b** [comment]
@severinlandolt Hey, can you see if the changes work?

**Comment by severinlandolt** [comment]
Thanks! Will be taking a look later this week!

**Comment by qasimsiddiqui** [comment]
Hey, I came across this PR when I faced the issue of a duplicate tailwind.config.ts file. Thanks for the PR. Hope it gets merged soon. 

A small suggestion: 
Do we need to ask for user input again to update the tailwind.config file? What if we change the initial message to `"Running this command will install dependencies and tailwind configurations. Proceed?"` What do you think?

@rajdip-b @severinlandolt

File: src/index.ts [hunk 1]
```diff
@@ -69,7 +69,7 @@ async function main() {
           type: "confirm",
           name: "proceed",
           message:
-            "Running this command will install dependencies and overwrite your existing tailwind.config.js. Proceed?",
+            "Running this command will install dependencies and. Proceed?",
```
Comment (qasimsiddiqui) [inline src/index.ts:72]: There is an extra `and.`

It should be `"Running this command will install dependencies. Proceed?"`

**Comment by rajdip-b** [comment]
hey @severinlandolt. sorry man! i was away for a long time. I will get to this issue within a few hours and update the PR. hope this gets merged then. Also, @qasimsiddiqui, went through your suggestions. I will implement what I can!

**Comment by rajdip-b** [comment]
> Hey, I came across this PR when I faced the issue of a duplicate tailwind.config.ts file. Thanks for the PR. Hope it gets merged soon.
> 
> A small suggestion: Do we need to ask for user input again to update the tailwind.config file? What if we change the initial message to `"Running this command will install dependencies and tailwind configurations. Proceed?"` Asking user input again seems to be redundant. What do you think?
> 
> @rajdip-b @severinlandolt

From what I can understand, yes, it is a good idea to use the message you proposed. Additionally, I have extended the message so that it's a bit more elaborate. `"Running this command will install dependencies and overwrite existing tailwind configurations. Proceed?"`

**Comment by rajdip-b** [comment]
@severinlandolt can you give it a review?

**Pull request merged**